tests/internal/scripts/ccuda/cuda_array_variable.py

```diff
@@ -0,0 +1,19 @@
+# pylint: disable=missing-function-docstring, disable=unused-variable, missing-module-docstring/
+
+from pyccel.decorators import kernel, types
+from pyccel import cuda
+
+@kernel
+@types('int[:]')
+def square(a):
+    index = cuda.blockIdx(0) * cuda.blockDim(0) + cuda.threadIdx(0)
+    a[index] = a[index] * a[index]
+
+if __name__ == '__main__':
+    threads_per_block = 5
+    n_blocks = 1
+    arr_var = [0,1,2,3,4]
```

Comment:
Please try to use officially supported syntax in tests. Here you can use a tuple.

```suggestion
    arr_var = (0,1,2,3,4)
```

Reply:
Does that mean that i should replace all lists with tuples, or only the ones assigned to variables ?

Reply:
Changed it to tuple here [b8643d2](https://github.com/pyccel/pyccel/pull/1236/commits/b8643d26a6fe5c53a8fe34eb79d67b376d89109c).

Reply:
Generally tuples should be used instead of lists, but we tolerate lists when there is a strong convention (e.g `np.array([1,2,3])`, it is very rare to use `np.array((1,2,3))` so it may be harder to recognise immediately)